tests/debugging/test_config.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from ddtrace.debugging._config import DEFAULT_PROBE_API_URL
+from ddtrace.debugging._config import DebuggerConfig
+from ddtrace.internal.utils.formats import parse_tags_str
+from ddtrace.version import get_version
+from tests.utils import override_env
+
+
+@pytest.mark.parametrize(
+    "dd_site, probe_api_url",
+    [
+        ("datadoghq.com", "https://app.datadoghq.com"),
+        ("datadoghq.eu", "https://app.datadoghq.eu"),
+        ("", DEFAULT_PROBE_API_URL),
+    ],
+)
+def test_probe_api_url(dd_site, probe_api_url):
+    with override_env(dict(DD_SITE=dd_site)):
+        DebuggerConfig().probe_api_url == probe_api_url
+
+
+def test_tags():
```

Comment:
Maybe we should add a test for service name to make sure that's correct when set?